src/Pages/2DPrinting/2DPrinting.js

```diff
@@ -194,6 +191,129 @@ export default function Printing(props) {
     setStatusModal(true);
   }
 
+  const printPageModalProps = {
+    isOpen: previewModal,
+    toggle: () => {
+      setPreviewModal(!previewModal);
+    },
+    size: 'xl',
+    ModalHeader: {
```

Comment:
will most of this stuff ever change? you could just keep it in the component instead of sending it over.

Reply:
It works and prints. The styling components might change if we are to enhance the UI of this page (which we should do), but the functionalities won’t (cases where state variables are passed). I like keeping these together just in case we need to link events/styling together in the future.